env0/provider.go

```diff
@@ -127,7 +127,13 @@ func configureProvider(version string, p *schema.Provider) schema.ConfigureConte
 		if err != nil {
 			return nil, diag.Diagnostics{diag.Diagnostic{Severity: diag.Error, Summary: err.Error()}}
 		}
+		apiClient := client.NewApiClient(httpClient)
 
-		return client.NewApiClient(httpClient), nil
+		// organizations fetched to cache Auth0 API response.
+		_, organizationError := apiClient.OrganizationIdCaching()
+		if organizationError != nil {
```

Comment:
in general error variables are named `err`.
This is the syntax you want to use:
```
if _, err := apiClient.OrganizationIdCaching(); err != nil {
   return nil, diag.Diagnostics{diag.Diagnostic{Severity: diag.Error, Summary: err.Error()}}
}
```

Reply:
thanks 
[156b5a5](https://github.com/env0/terraform-provider-env0/pull/456/commits/156b5a5da71d5a61a2537beeadda4af499586fa4)